earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -171,14 +171,27 @@ AssetVersionImplD::SetState(AssetDefs::State newstate, bool propagate)
     notify(NFY_DEBUG, "SetState: %s %s",
            ToString(newstate).c_str(),
            GetRef().c_str());
+    notify(NFY_VERBOSE, "newstate: %s not equal to current state: %s", 
+           ToString(newstate).c_str(), 
+           ToString(state).c_str());
     AssetDefs::State oldstate = state;
+    notify(NFY_VERBOSE, "Set oldstate: %s to current state: %s", 
+           ToString(oldstate).c_str(), 
+           ToString(state).c_str());
     state = newstate;
+    notify(NFY_VERBOSE, "Set current state: %s to newstate: %s", 
+           ToString(state).c_str(), 
+           ToString(newstate).c_str());
     try {
       // NOTE: This can end up calling back here to switch us to
       // another state (usually Failed or Succeded)
+      notify(NFY_VERBOSE, "Trying OnStateChange(%s, %s)", 
```

Comment:
I would consider removing this notify, it isn't really adding any information we don't already have